src/components/shared/pcsGrid/pcsGrid.js

```diff
@@ -81,6 +99,9 @@ export class PcsGrid extends Component {
 
   /** When a row is hard selected, try to fire a hard select event, plus any props callbacks */
   onSelectionChanged = () => {
+    const currentHardSelectIds = this.gridApi.getSelectedRows().map(({ id }) => ({ id }));
```

Comment:
This isn't actually going to work since this breaks the generality of the PcsGrid. The grid can't have any dependency on the shape of the data.